src/scraper.rs

```diff
@@ -1,27 +1,15 @@
-use std::thread::{spawn, JoinHandle};
-use std::thread;
-use std::time::Duration;
+use std::thread::JoinHandle;
 
 use chrono::{DateTime, Utc};
 
 use config::{CONFIG, GH_ORGS};
 use github;
 
-pub fn start_scraping() -> JoinHandle<()> {
-    // spawn the github scraper in the background
-    spawn(|| {
-        let sleep_duration = Duration::from_secs(CONFIG.github_interval_mins * 60);
-        loop {
-            match github::most_recent_update() {
-                Ok(gh_most_recent) => scrape_github(gh_most_recent),
-                Err(why) => error!("Unable to determine most recent GH update: {:?}", why),
-            }
-            info!("GitHub scraper sleeping for {} seconds ({} minutes)",
-                  sleep_duration.as_secs(),
-                  CONFIG.github_interval_mins);
-            thread::sleep(sleep_duration);
-        }
-    })
+pub fn start_scraping() -> Option<JoinHandle<()>> {
```

Comment:
Note to self (i.e. please ignore for PR review purposes), cc https://github.com/Centril/rfcs/pull/17.
This would have been so nice with:
```rust
pub fn start_scraping() -> Option<JoinHandle<()>> = try {
    spawn_thread("GitHub scraper", CONFIG.github_interval_mins?, || try {
        scrape_github(github::most_recent_update()?);
    })
};
```